docs/modules/getting-started/pages/get-started-docker.adoc

```diff
@@ -313,23 +313,49 @@ go get github.com/hazelcast/hazelcast-go-client
 +
 [source,go]
 ----
-import "github.com/hazelcast/hazelcast-go-client"
+package main
+
+import (
+  "context"
+  "fmt"
+  "log"
 
-func mapSampleRun() {
-  cb := hazelcast.NewConfigBuilder()
+  "github.com/hazelcast/hazelcast-go-client"
+)
 
-  cb.Cluster().SetName("hello-world") <1>
+func main() {
+  ctx := context.TODO()
 
-  client, err := hazelcast.StartNewClientWithConfig(cb) <2>
+  cb := hazelcast.NewConfig()
+  cb.ClientName = "hello-world" <1>
 
-	mp, _ := client.GetMap("my-distributed-map") <3>
+  hz, err := hazelcast.StartNewClientWithConfig(ctx, cb) <2>
+  if err != nil {
+    log.Fatal(err)
+  }
+
+  mp, err := hz.GetMap(ctx, "my-distributed-map") <3>
+  if err != nil {
+    log.Fatal(err)
+  }
 
   <4>
-	mp.Put("1", "John")
-	mp.Put("2", "Mary")
-	mp.Put("3", "Jane")
+  _, err = mp.Put(ctx, "1", "John")
```

Comment:
It's a bit confusing to use numbers as strings. How about using numbers directly or some other string, like: `"person1"`.